src/Initialize.mss

```diff
@@ -48,6 +48,22 @@ function InitGlobals (extensions) {
         'C', 'cut'
     );
 
+    Self._property:BarlineTypeMap = CreateSparseArray();
+    BarlineTypeMap[SpecialBarlineStartRepeat] = 'rptstart';
+    BarlineTypeMap[SpecialBarlineEndRepeat] = 'rptend';
+    BarlineTypeMap[SpecialBarlineDashed] = 'dashed';
+    BarlineTypeMap[SpecialBarlineDouble] = 'dbl';
+    BarlineTypeMap[SpecialBarlineFinal] = 'end';
+    BarlineTypeMap[SpecialBarlineInvisible] = 'invis';
+    BarlineTypeMap[SpecialBarlineNormal] = 'single';
+    BarlineTypeMap[SpecialBarlineDotted] = 'dotted';
+    BarlineTypeMap[SpecialBarlineThick] = 'heavy';
+    // no MEI equiv:
+    // BarlineTypeMap[SpecialBarlineBetweenStaves] = ' ';
+    // BarlineTypeMap[SpecialBarlineTick] = ' ';
+    // BarlineTypeMap[SpecialBarlineShort] = ' ';
```

Comment:
This would go into [`@bar.method`](https://music-encoding.org/guidelines/v4/data-types/data.barmethod.html).

Reply:
Ah, right, I changed how the attributes are added.